src/pages/home/models-list/models-list.style.js

```diff
@@ -11,7 +11,7 @@ export const useStyles = makeStyles((theme) => ({
     justifyContent: 'center',
     transition: 'height 1s ease',
     '@media (max-width: 568px)': {
-      maxHeight: '600px'
+      maxHeight: '1000px'
```

Comment:
Did you check the behaviour of this block on different devices?

Reply:
Yes. I checked this in different display resolution and everything was ok.

Reply:
Good job